### PR TITLE
use docker

### PR DESCRIPTION
**Pull request by Zhima-Mochi**
We can use [Docker](https://www.docker.com/) if we don't want to deal with setting up the environment.

### CODE REVIEW

**Comment by yihong0618** [comment]
will take a look tomorrow.

**Comment by kenwoodjw** [comment]
maybe I can review this pr

**Comment by yihong0618** [comment]
> maybe I can review this pr

thanks

File: README.md [hunk 1]
```diff
@@ -39,12 +39,43 @@ export OPENAI_API_KEY=${your_api_key}
...
+You can use [Docker](https://www.docker.com/) if you don't want to deal with setting up the environment.
+```shell
+# build image
+docker image build --tag bilingual_book_maker .
+
+# run container
+# "$folder_path" represents the folder where your book file is located. Also, it is where the processed file will be stored.
+
+# Windows PowerShell
+$folder_path=your_folder_path
```
Comment (kenwoodjw) [inline README.md:58]: maybe need a tip:  absolute path

File: README.md [hunk 2]
```diff
@@ -39,12 +39,43 @@ export OPENAI_API_KEY=${your_api_key}
...
+$book_name=your_book_name
+$openai_key=your_api_key
+$language=your_language
+
+docker container run --rm --name bilingual_book_maker --mount type=bind,source=$folder_path,target='/app/test_books' bilingual_book_maker --book_name "/app/test_books/$book_name" --openai_key $openai_key --no_limit --language $language
```
Comment (kenwoodjw) [inline README.md:63]: docker run is common use

File: README.md [hunk 3]
```diff
@@ -39,12 +39,43 @@ export OPENAI_API_KEY=${your_api_key}
...
 python3 make_book.py --book_name 'animal_farm.epub' --openai_key sk-XXXXX --api_base 'https://xxxxx/v1'
 # or
 python make_book.py --book_name 'animal_farm.epub' --openai_key sk-XXXXX --api_base 'https://xxxxx/v1'
 ```
 
+## Docker
+You can use [Docker](https://www.docker.com/) if you don't want to deal with setting up the environment.
+```shell
+# build image
+docker image build --tag bilingual_book_maker .
```
Comment (kenwoodjw) [inline README.md:52]: docker build is common use

Comment (Zhima-Mochi) [reply README.md]: update

File: README.md (hunk 1 above)
Comment (Zhima-Mochi) [reply README.md]: update

File: README.md (hunk 2 above)
Comment (Zhima-Mochi) [reply README.md]: update

Comment (kenwoodjw) [reply README.md]: @yihong0618  I test everything is fine, can merge

**Comment by yihong0618** [comment]
@kenwoodjw @Zhima-Mochi Thanks

**Pull request merged**